src/Lykke.Service.CandlesHistory.Core/Domain/Candles/MarketType.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Lykke.Service.CandlesHistory.Core.Domain.Candles
+{
+    public enum MarketType
+    {
+        Spot,
+        Mt
```

Comment:
Give a full name

Reply:
MT is a commonly-used term in Lykke. Could you explain with more details why we needed to get it renamed here?

Reply:
Not everyone knows this abbreviation. But it is minor.